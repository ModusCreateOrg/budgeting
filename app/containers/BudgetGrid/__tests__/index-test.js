import React from 'react';
import renderer from 'react-test-renderer';
import { BudgetGrid } from '../index';

// mock nested component
jest.mock('containers/EntryFormRow');

it('renders correctly', () => {
  const tree = renderer.create(<BudgetGrid />).toJSON();
  expect(tree).toMatchSnapshot();
});
