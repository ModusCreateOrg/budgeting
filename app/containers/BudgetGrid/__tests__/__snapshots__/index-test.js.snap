// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<table
  className="budgetGrid"
>
  <thead>
    <tr>
      <th>
        Category
      </th>
      <th>
        Description
      </th>
      <th>
        Amount
      </th>
    </tr>
  </thead>
  <tbody />
  <tfoot />
</table>
`;
